### DISCUSSION TITLE
First Blood

### DISCUSSION THREAD

**Discussion by C1sterna** [fraxiinus/ReplayBook#240] [Feature Request]
Can u add to statistics:
firstBloodAssist
firstBloodKill
firstTowerAssist 
firstTowerKill
?

**Comment by fraxiinus** [comment]
Unfortunately first blood data is not available in the replay metadata.

**Reply by gabriellimo** [reply]
what about champion bans?

**Reply by fraxiinus** [reply]
Also not available. The full model of fields can be viewed here: https://github.com/fraxiinus/roflxd.cs/blob/master/Rofl.Extract.Data/Models/Rofl/PlayerStats.cs

The full model is also available in ReplayBook's exporter.
